ts/Grid/Core/Defaults.ts

```diff
@@ -72,7 +72,8 @@ namespace Defaults {
                 bufferSize: 10,
                 minVisibleRows: 2,
                 strictHeights: false,
-                virtualization: true
+                virtualization: false,
+                threshold: 50
```

Comment:
If we introduce threshold with default value, virtualization should be `undefined`. If virtualization is defined, then threshold should be ignored.

Reply:
It works this way in init, I'm removing it from the defaults, thanks!